pangeo_forge_esgf/dynamic_kwargs.py

```diff
@@ -96,21 +98,29 @@ async def response_data_processing(
     session: aiohttp.ClientSession,
     response_data: List[Dict[str, str]],
     iid: str,
+    ssl: ssl.SSLContext = None,
 ) -> Tuple[List[str], Dict[str, Dict[str, str]]]:
 
-    table_id = facets_from_iid(iid).get("table_id")
+    # table_id = facets_from_iid(iid).get("table_id")
     urls = [r["url"] for r in response_data]
     sizes = [r["size"] for r in response_data]
     titles = [r["title"] for r in response_data]
 
+    # determine frequency
+    frequencies = [r["frequency"][0] for r in response_data]
+    # check if not all files have the same frequency, that should never happen.
+    if not all(f == frequencies[0] for f in frequencies):
+        raise ValueError("Determined frequencies are not all equal.")
+    frequency = frequencies[0]
+
```

Comment:
I added `frequency` (or `time_frequency`) to the filtered response. This should also work for `obs4MIPs` or `input4MIPs`...